internal/api/api.go

```diff
@@ -148,6 +150,29 @@ type ContainerCatalogEntry struct {
 		TopLayerID             string `json:"top_layer_id"`
 		UncompressedTopLayerID string `json:"uncompressed_top_layer_id"`*/
 }
+type ChartStruct struct {
+	Entries map[string][]struct {
+		Name        string `json:"name"`
+		Version     string `json:"version"`
+		KubeVersion string `json:"kubeVersion"`
+	} `json:"entries"`
+}
```

Comment:
you can unmarshall yaml to this struct directly without passing by JSON transformation. 
``
yaml.Unmarshal()
``